### PR TITLE
Project ID removes spaces (Issue #467)

### PR DESCRIPTION
**Pull request by ubionexd**
No more annoying spaces, numbers or special characters.
The way i'we done it is probably not the best, but it works!
Fixed GitHub Issue #467

### CODE REVIEW

**Comment by wafflecoffee** [comment]
This field accepts slugs, though, and slugs can have other characters (e.g. fabric-api or lovely_snails).

File: pages/_type/_id/version.vue [hunk 1]
```diff
@@ -398,6 +398,7 @@
             <input
               v-model="newDependencyId"
               type="text"
+              oninput="this.value=this.value.replace(/[^a-z]/gi,'')"
```
Comment (Prospector) [inline pages/_type/_id/version.vue:401]: Should only remove whitespace

**Review by Prospector — CHANGES_REQUESTED** [review]

**Comment by ubionexd** [comment]
I will change it as soon as i can, but im off pc rn. Its only 10 AM.

File: pages/_type/_id/version.vue (hunk 1 above)
Comment (ubionexd) [reply pages/_type/_id/version.vue]: Pushed a commit to this pr. Should be good now.

**Review by wafflecoffee — APPROVED** [review]

File: pages/_type/_id/version.vue [hunk 2]
```diff
@@ -398,6 +398,7 @@
             <input
               v-model="newDependencyId"
               type="text"
+              oninput="this.value = this.value.replace(' ', '')"
```
Comment (Prospector) [inline pages/_type/_id/version.vue:401]: Should we maybe use .trim() so it gets any whitespace?

**Review by Prospector — APPROVED** [review]

**Pull request merged**